packages/backend/src/datasource/datasource.service.spec.ts

```
@@ -117,14 +118,17 @@ describe('DataSourceService', () => {
       };
     });
 
-    jest
+    const createClient = jest
       .spyOn(service as unknown as { createClient: () => unknown }, 'createClient')
```

ESLint

[error] Replace `service·as·unknown·as·{·createClient:·()·=>·unknown·},·'createClient'` with `⏎········service·as·unknown·as·{·createClient:·()·=>·unknown·},⏎········'createClient',⏎······`

(prettier/prettier)